packages/components/BREAKING_CHANGES.md

```diff
@@ -192,6 +229,231 @@ Remove `priority-menu-label` from all `<syn-prio-nav>` elements on the page. Thi
 
 ---
 
+### `<syn-side-nav>`
+
+#### ⚠️ Adjusted the open width of the side-nav
+
+**Associated Ticket(s)**:
+
+- [#479](https://github.com/synergy-design-system/synergy-design-system/issues/479)
+
+**Reason**:
+
+To better fit in case of mobile versions / smaller screen sizes, we decided to adjust the width of the opened `<syn-side-nav/>`, to be a bit narrower from 400px to 320px.
+
+**Migration Steps**:
+If for some reason the old width of 400px are desired, the css property `--side-nav-open-width` of the side navigation can be overwritten like shown in the following example.
```

Comment:
Do we now propagate the width somewhere? If yes, we should communicate this here.